app/src/layouts/common/appSidebar/appSidebar.scss

```diff
@@ -121,6 +121,14 @@
 
   &:hover {
     background-color: $COLOR--darkmode-gray-500;
+
+    button div {
+      color: $COLOR--white-two;
+
+      path {
+        fill: $COLOR--white-two;
+      }
+    }
   }
 }
```

Comment:
Looks as a duplicate of lines 135-147.
Can in be handled via single set of such styles?

Another comment is about using relative tags selectors (button > div).
Can we avoid them here in favor of classes to less depend on child components HTML structure?

Reply:
1 Done.
2 These changes will be in the component from the component library, so it's better to leave it that way.